### DISCUSSION TITLE
Customizing OpenThread Web GUI page running on RPI

### DISCUSSION THREAD

**Discussion by nikhil30081995** [openthread/openthread#8439] [Q&A] [answered]
Hello, we have a OTBR setup running on RPi 3B+. As we are using RPi, we want our customers to connect their OTBR to their own WIFI network. So for this purpose, we would like to provide them a Web GUI interface, which allow them to do that. 

So i would like t know where i can find the OT WEB GUI page (html etc) in RPi, so that i can customize it a little bit for WiFi connection setup ?

**Comment by jwhui** [comment, marked as answer]
https://github.com/openthread/ot-br-posix/tree/main/src/web

**Reply by nikhil30081995** [reply]
> https://github.com/openthread/ot-br-posix/tree/main/src/web

Thanks for information. 

I would like to know if i edit the home page contents where information regarding open thread is explained and customize it to form a page for entering WIFI credentials such as SSID and Password.  Will it effect open thread working any way ?

Below is the page i am referring too: 
![image](https://user-images.githubusercontent.com/84308431/203700044-1bda10e4-b5ae-4044-b2d4-c69a68daabfe.png)

**Reply by jwhui** [reply]
What you are showing is just informational text. It does not have any effect on the Thread protocol operation.
